src/test/java/conan/testUtils/OpenSSLProjectImpl.java

```diff
@@ -135,7 +135,7 @@ public PicoContainer getPicoContainer() {
     @NotNull
     @Override
     public MessageBus getMessageBus() {
-        return new MessageBusImpl.RootBus("");
+        return ApplicationManager.getApplication().getMessageBus();
```

Comment:
I actually have no idea if this is valid.

However, tests pass and we don't seem to call this method in tests.

I think it's unlikely this would cause false positives in tests, so I think it's OK.